c-pallets/file-bank/src/lib.rs

```diff
@@ -743,10 +753,10 @@ pub mod pallet {
 		/// Parameters:
 		/// - `file_hash`: Invalid file hash
 		#[transactional]
-		#[pallet::weight(22_777_000)]
+		#[pallet::weight(22_777_000)] // harcoded weigth?
 		pub fn clear_invalid_file(origin: OriginFor<T>, file_hash: Vec<u8>) -> DispatchResult {
 			let sender = ensure_signed(origin)?;
-			let bounded_string: BoundedString<T> =
+			let bounded_string: BoundedString<T> = // Can be used Self::vec_to_bound instead?
```

Comment:
Based on your suggestion, I chose to use vec_to_bound:
https://github.com/CESSProject/cess/pull/88/files#diff-9c37b1a61a6e9394aac1e9383bf9d3fac9539378d1af8fcd75aed4f8a4f42967R769